### DISCUSSION TITLE
How to install `parcel` cli (parcel 2)?

### DISCUSSION THREAD

**Discussion by saona-raimundo** [parcel-bundler/parcel#8423] [Q&A] [answered]
Sorry if it sounds stupid, but I can only find how to add parcel dependency to a project (using yarn or npm)... but I can not find how to install the parcel cli!

I would expect to find it in one of the following locations (but is not):
- https://parceljs.org/
- https://parceljs.org/docs/
- https://parceljs.org/features/cli/

The cli is the fastest way to get started with parcel, but there is no instructions on how to install it.

Thanks beforehand!

**Comment by mischnic** [comment, marked as answer]
The `parcel` npm package contains the `parcel` CLI

**Reply by saona-raimundo** [reply]
Thank you for your answer. Do you mean the following?

1. There is NO WAY to install a global command `parcel` that is available in PATH.
2. To use `parcel`, it has to be done through `npm` or `yarn`.
3. To call the `parcel` cli, one needs to follow the following steps:
  -  Install `npm` and `npx` if you have not done before.
  - Run `npm install parcel`.
    + This will create three assets in the path where this command was run: (1) `package.json`; (2) `node_modules`; (3) `package-lock.json`
  - Run `parcel` by calling `npx parcel`.

This installation is local to the path in which you run `npm install parcel` in the sense that `npx parcel` will be available in all subfolders.

If this is correct, in the end, THERE IS NO `parcel` CLI, only a binary node module, right?

**Reply by mischnic** [reply]
> There is NO WAY to install a global command parcel that is available in PATH.

You can do `yarn global add parcel` or `npm install -g parcel`. But I don't recommend it

> To use parcel, it has to be done through npm or yarn.

Yes

> To call the parcel cli, one needs to follow the following steps:

Either that or install it globally via npm/Yarn.

> If this is correct, in the end, THERE IS NO parcel CLI, only a binary node module, right?

There is no single native binary executable if that's what you mean.
